website/src/components/UsersCell.tsx

```diff
@@ -11,33 +11,68 @@ import {
   Th,
   Thead,
   Tr,
+  useToast,
 } from "@chakra-ui/react";
 import Link from "next/link";
 import { useState } from "react";
 import { get } from "src/lib/api";
 import type { User } from "src/types/Users";
 import useSWR from "swr";
 
+interface Pagination {
+  /**
+   * The user's `display_name` used for pagination.
+   */
+  cursor: string;
```

Comment:
Now all users are coming from the real backend and the data is combined with our prisma db, correct? why are we using `display_name`? Is it a unique key for the user?

Reply:
This is due to the backend API.  `/api/v1/frontend_users/` takes in a `gte` or a `le` field for pagination and it sorts (filters actually) based on `display_name`.  I had to read `backend/oasst_backend/user_repository.py` to verify this and hence why I also updated it to make the ordering non-inclusive.

Reply:
I see...